### ISSUE TITLE
Implementation details of method c (Feature-clone)

### ISSUE THREAD

**Issue by zjwzcx** [revisitq/Pseudo-Stereo-3D#2]
Hi! Thanks for your awesome work!

In Sec. 4.4, you mentioned 'clone the left features as the virtual right features'. Does this mean that the **only difference** between Model C (Feature-clone) and LIGA-Stereo is that Model C replaces right images with cloned left images?

Look forward to your answer! @revisitq @hang3DV

**Comment by revisitq** [comment]
> Hi! Thanks for your awesome work!
> 
> In Sec. 4.4, you mentioned 'clone the left features as the virtual right features'. Does this mean that the **only difference** between Model C (Feature-clone) and LIGA-Stereo is that Model C replaces right images with cloned left images?
> 
> Look forward to your answer! @revisitq @hang3DV
As shown in Table 6(supplementary file), we are not replace right images with cloned left images to do feature-clone. We clone the output left features of the SPP module as the right features. Please look forward to our code, thank you!

**Comment by zjwzcx** [comment]
> 

Thanks for your reply! I made a typo (left image -> left feature) in my question... 

I tried to insert `right_stereo_feat = left_stereo_feat.clone()` in LIGA-Stereo's code, which was the only change I made. However, there is still a big performance gap between my experimental result and the result of Exp 5&6 in Table 1. I am confused about this. 

When will you guys release your code? I'm really looking forward to that!
